src/neml2/drivers/ModelDriver.cxx

```diff
@@ -146,4 +177,12 @@ ModelDriver::diagnose() const
   Driver::diagnose();
   neml2::diagnose(*_model);
 }
+
+void
+ModelDriver::to(Device dev)
+{
+  _device = dev;
+  setup();
+}
```

Comment:
Why do we need this method as well as `TransientRegression::to`?

Reply:
Oh I added this... now I'm questioning myself. We need this to run regression tests on different devices.